### PR TITLE
 solve failing to create blb during ensureLoadBalancer issue#111

### PR DESCRIPTION
**Pull request by wccsama**
solve failing to create blb during ensureLoadBalancer issue#111

### CODE REVIEW

File: pkg/cloud-provider/load_balancer_blb.go [hunk 1]
```diff
@@ -180,6 +180,9 @@ func (bc *Baiducloud) waitForLoadBalancer(lb *blb.LoadBalancer) (*blb.LoadBalanc
 		if !exist {
 			glog.V(3).Infof("getBCELoadBalancer not exist: %s, retry", lb.BlbId)
 			if index >= 9 {
+				if service.Annotations != nil {
+					delete(service.Annotations, ServiceAnnotationCceAutoAddLoadBalancerId)
+				}
```
Comment (tianyuansun) [inline pkg/cloud-provider/load_balancer_blb.go]: 定义 NotFound error，return 该 error，不在该函数内处理 delete()，外部 defer 处理可能更好一点

**Review by tianyuansun — COMMENTED** [review]
delete 在外层处理

**Comment by tianyuansun** [comment]
fix: https://github.com/baidu/cloud-provider-baiducloud/issues/111

**Review by hello2mao — APPROVED** [review]

**Review by tianyuansun — APPROVED** [review]
/lgtm

**Pull request merged**